tests/FSharpLint.Benchmarks/Benchmark.fs

```diff
@@ -23,7 +23,7 @@ type Benchmark () =
 
         parseResults.ParseTree
 
-    let (</>) x y = Path.Combine(x, y)
+    let (</>) basePath relativePath = Path.Combine(basePath, relativePath)
```

Comment:
@Mersho let's add a DRY-refactoring commit removing this? I think I've seen it somewhere else